riptide-problem/src/main/java/org/zalando/riptide/problem/ProblemRoute.java

```diff
@@ -10,17 +14,43 @@
 
 public final class ProblemRoute {
 
-    private static final Route ROUTE = dispatch(contentType(),
-            on(parseMediaType("application/problem+json")).call(Exceptional.class, Exceptional::propagate),
-            on(parseMediaType("application/x.problem+json")).call(Exceptional.class, Exceptional::propagate),
-            on(parseMediaType("application/x-problem+json")).call(Exceptional.class, Exceptional::propagate));
+    private static final MediaType PROBLEM = parseMediaType("application/problem+json");
```

Comment:
So you may just hardcode them into `problemHandling(final ThrowingConsumer<Exceptional, ? extends Exception> consumer)`